Debug-frontend-tests.md

```diff
@@ -4,6 +4,8 @@
 * [Find tests](#find-tests)
 * [Isolate tests](#isolate-tests)
 * [Verbose mode](#verbose-mode)
+* [Downloading the combined-tests.spec.js file](#downloading-the-combined-tests.spec.js-file)
```

Comment:
This link doesn't work. You can try it out here: https://github.com/oppia/oppia-web-developer-docs/blob/2e86b515a9d7db2b1a9eafbe13ed3f569b4cb7c8/Debug-frontend-tests.md

Reply:
Fixed